packages/ember-application/lib/system/engine-instance.js

```diff
@@ -220,13 +215,12 @@ EngineInstance.reopenClass({
     registry.injection('view', '_environment', '-environment:main');
     registry.injection('route', '_environment', '-environment:main');
 
+    let inject = '-inert';
```

Comment:
This is a good change, but lets give the variable a better name.  In context, I would suggest `rendererType`.